parquet/api.py

```diff
@@ -77,6 +73,7 @@ def _parse_header(self, f, verify=True):
         for i, rg in enumerate(self.row_groups):
             for chunk in rg.columns:
                 self.group_files.setdefault(i, set()).add(chunk.file_path)
+        self.helper = schema.SchemaHelper(self.schema)
```

Comment:
This name seems generic to me.  I see in this line that it has something to do with the schema but I suspect it'll be more ambiguous when seen elsewhere.